sklearn/ensemble/tests/test_gradient_boosting.py

```diff
@@ -1196,16 +1196,16 @@ def test_gradient_boosting_early_stopping():
 
     # Without early stopping
     gbc = GradientBoostingClassifier(
-        n_estimators=100, learning_rate=0.1, max_depth=3, random_state=42
+        n_estimators=5, learning_rate=0.1, max_depth=3, random_state=42
```

Comment:
For the test to be meaningful we, need to use significantly more than 28 trees here. Let's do 50:

```suggestion
        n_estimators=50, learning_rate=0.1, max_depth=3, random_state=42
```